src/components/v5/common/AvatarUploader/partials/ErrorContent.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { FC } from 'react';
+import { useIntl } from 'react-intl';
+import clsx from 'clsx';
+
+import Icon from '~shared/Icon';
+import { DropzoneErrors } from '~shared/AvatarUploader/helpers';
+import { TextButton } from '~v5/shared/Button';
+import styles from './AvatarUploader.module.css';
+import { ErrorContentProps } from '../types';
+
+const displayName = 'v5.common.AvatarUploader.partials.ErrorContent';
+
+const ErrorContent: FC<ErrorContentProps> = ({
+  errorCode,
+  handleFileRemove,
+  open,
+  getInputProps,
+  fileRejections,
+}) => {
+  const { formatMessage } = useIntl();
+
+  const errorMessage =
+    (errorCode === DropzoneErrors.TOO_LARGE && 'too.large.file.error') ||
+    (errorCode === DropzoneErrors.CUSTOM && 'upload.failed') ||
+    ((errorCode === DropzoneErrors.INVALID && 'invalid.type.error') as string);
+
+  return (
+    <div
+      className={clsx(
+        'gap-3 bg-base-white border-negative-400',
+        styles.contentWrapper,
+      )}
+    >
+      <div className="w-[2.55rem] mb-[0.5rem]">
```

Comment:
Please, correct all custom sizes in [] in this file.
There are incorrect.
https://tailwindcss.com/docs/customizing-spacing

Reply:
Also, the DefaultContent and the ErrorContent have very similar HTML and styles. I think it should be one component with a prop that controls default and error states.